vm/vmExpr.go

```diff
@@ -780,3 +791,44 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 		return nilValue, newStringError(e, "unknown expression")
 	}
 }
+
+// exprBoolValue attempts to evaluate the  true/false value of an expression.
+// When converting a string, the function returns
+// true if the string is the word "true", false if the string is "false",
+// false if the string is 0 or 0.0 and true otherwise if the string is nonempty.
+// The 'caseSensitive' flag affects the behavior when converting strings;
+// if set to true, the function only matches on "true" or "false", not
+// e.g. "True" or "FALSE"
+func exprBoolValue(v reflect.Value) bool {
+	if v.Kind() == reflect.Ptr || v.Kind() == reflect.Interface {
+		v = v.Elem()
+	}
+	switch v.Kind() {
+	case reflect.Float64, reflect.Float32:
+		return v.Float() != 0
+	case reflect.Int64, reflect.Int32, reflect.Int16, reflect.Int8, reflect.Int:
+		return v.Int() != 0
+	case reflect.Bool:
+		return v.Bool()
+	case reflect.String:
+		s := v.String()
+		if v.Len() == 0 {
+			return false
+		}
+		if s == "true" {
```

Comment:
For true, can it be any of the following: 1, t, T, TRUE, true, True
Like ParseBool:
https://golang.org/pkg/strconv/#ParseBool